package.json

```diff
@@ -76,7 +76,7 @@
     "ui:build:css": "sass --source-map --precision=8 ./src/clr-angular/main.scss ./dist/clr-ui/clr-ui.css; sass --source-map --precision=8 ./src/clr-angular/dark-theme.scss ./dist/clr-ui/clr-ui-dark.css",
     "ui:build:prefix": "postcss ./dist/clr-ui/clr-ui.css -o ./dist/clr-ui/clr-ui.css; postcss ./dist/clr-ui/clr-ui-dark.css -o ./dist/clr-ui/clr-ui-dark.css",
     "ui:build:src": "cpy --parents --cwd='src/clr-angular/' '**/*.scss' ../../dist/clr-ui/src/",
-    "ui:build:optimize": "csso -i ./dist/clr-ui/clr-ui.css -o ./dist/clr-ui/clr-ui.min.css -s file; csso -i ./dist/clr-ui/clr-ui-dark.css -o ./dist/clr-ui/clr-ui-dark.min.css -s file",
+    "ui:build:optimize": "csso -i ./dist/clr-ui/clr-ui.css -o ./dist/clr-ui/clr-ui.min.css -s file --no-restructure; csso -i ./dist/clr-ui/clr-ui-dark.css -o ./dist/clr-ui/clr-ui-dark.min.css -s file --no-restructure",
```

Comment:
Should probably apply this rule to other `csso` calls in the other scripts?

Reply:
good call. i thought it was only here. i was in too much of a hurry. i'll fix that.

Reply:
Updated. Thanks for catching that!